specs/PushNotifications/ToastNotifications-spec.md

```diff
@@ -0,0 +1,567 @@
+    };
+
+    [experimental]
+    // Event args for the Notification Activation
+    runtimeclass AppNotificationActivatedEventArgs
+    {
+        // Arguments from the invoked button. Empty for Default Activation with no launch args specified in payload.
+        String ActivationArgs{ get; };
+
+        // The data from the input elements of a Notification like a TextBox
+        Windows.Foundation.Collections.IMapView<String, String> UserInput{ get; };
+    };
+
+    [experimental]
+    // Notification Progress Data
+    runtimeclass AppNotificationProgressData
+    {
+        // Initializes a new Instance of NotificationProgressData
+        AppNotificationProgressData();
+
+        // Gets or sets the sequence number of this notification data.
+        // When multiple NotificationProgressData objects are received, the system displays the data with the greatest non-zero number. 
+        UInt32 SequenceNumber;
+
+        // Gets/Sets the value for the title. Binds to {progressTitle} in progress xml tag.
+        String Title;
+
+        // Gets/Sets the Value for the numerical Progress percentile: a number between 0 and 1. Binds to {progressValue} in progress xml tag.
+        Double Value;
+
+        // Gets/Sets the Value for the Progress String. Binds to {progressValueString} in progress xml tag
+        String ValueStringOverride;
+
+        // Gets/Sets the Value for the Status. Binds to {progressStatus} in progress xml tag
+        String Status;
+    };
+
+    [experimental]
+    // The Notification User Setting or Notification Group Policy Setting
+    enum AppNotificationSetting
+    {
+        Enabled, // Notification is not blocked by settings or group policy
+        DisabledForApplication, // Notification is blocked by a user defined App Setting
+        DisabledForUser, // Notification is blocked by a user defined Global Setting
+        DisabledByGroupPolicy, // Notification is blocked by Group Policy
+        DisabledByManifest, // Notification is blocked by a setting in the manifest. Only for packaged applications.
+    };
+
+    [experimental]
+    // The Result for a Notification Progress related operation
+    enum AppNotificationProgressResult
+    {
+        Succeeded, // The progress operation succeeded
+        Failed, // The progress operation failed
+        AppNotificationNotFound, // The progress operation failed to find a Notification to process updates
+    };
+
+    [experimental]
+    enum AppNotificationPriority
+    {
+        Default, // The notification should have default behavior in terms of delivery and display priority during connected standby mode.
+        High, // The notification should be treated as high priority. For desktop PCs, this means during connected standby mode the incoming notification can turn on the screen for Surface-like devices if it doesn't have a closed lid detected.
+    };
+
+    [experimental]
+    // Represent a Notification Notification construct
```

Comment:
Notification repeated. best to just delete this comment as it add no new information.